lib/IDE/Refactoring.cpp

```diff
@@ -4327,12 +4337,26 @@ struct AsyncHandlerDesc {
     }
   }
 
+  /// If the async function returns a tuple, the label of the \p Index -th
+  /// element in the returned tuple. If the function doesn't return a tuple or
+  /// the element is unlabeled, an empty identifier is returned.
+  Identifier getAsyncReturnTypeLabel(size_t Index) const {
```

Comment:
Could just take the param here instead - `getAsyncReturnTypes` already has it and the other use in `addCompletionHandlerArgument` can just grab it.

Reply:
Not really, because if we have a completion handler `(_ param: String) -> Void` we can’t label the return type and I wanted to put that logic in a centralized place, that’s why I created the function.

Reply:
Ah I see, that makes sense... Another option is to calculate up front, but this is fine or now.

Reply:
I mean, there’s not much to “calculate” here.